Lib/encodings/aliases.py

```diff
@@ -318,6 +318,7 @@
     'iso_ir_157'         : 'iso8859_10',
     'l6'                 : 'iso8859_10',
     'latin6'             : 'iso8859_10',
+    'latin_6'            : 'iso8859_10',
```

Comment:
These aliases are not very common, but it may still be useful to add them to the aliases.py. Please open a separate issue / PR for this.